src/js/hooks/use-get-user.ts

```diff
@@ -1,27 +1,27 @@
-
 import { LOCAL_STORAGE_KEY_FOR_SESSION_TOKEN } from '@/constants';
 import { apiClient } from '@/components/FavouriteShabadButton/utils/api-client';
 import { getQueryParams } from '@/util';
 import { useQuery } from 'react-query';
 
 export const useGetUser = <D>() => {
   const { token } = getQueryParams();
-  const isUserLoggedIn =
-    !!(window.localStorage.getItem(LOCAL_STORAGE_KEY_FOR_SESSION_TOKEN) || token);
+  const isUserLoggedIn = !!(
```

Comment:
NIT: Can use this 

`userToken = window.localStorage.getItem(LOCAL_STORAGE_KEY_FOR_SESSION_TOKEN) || token`

`enabled: !!userToken`

Reply:
fixed